packages/@glimmer/application/src/loaders/runtime-compiler/loader.ts

```diff
@@ -52,4 +52,8 @@ export default class RuntimeCompilerLoader implements Loader {
       mainLayout.asLayout().compile()
     ));
   }
+
+  getComponentTemplateIterator(app: BaseApplication, env: Environment, builder: ElementBuilder, componentName: string, args: RenderComponentArgs): Promise<TemplateIterator> {
+    throw new Error("Method not implemented.");
```

Comment:
What's needed to support this?

Reply:
I unfortunately haven't had time to dig through all the details needed to implement this here. I was planning on addressing this as a separate issue once this PR was merged.